sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -133,24 +133,25 @@ def __init__(self, name_context, spec, counter_factory, state_sampler):
 
     # These are overwritten in the legacy harness.
     self.metrics_container = MetricsContainer(self.name_context.metrics_name())
-    self.scoped_metrics_container = ScopedMetricsContainer(
-        self.metrics_container)
+    self.scoped_metrics_container = ScopedMetricsContainer()
```

Comment:
Can this just be deleted?

Reply:
Right. Done.

Reply:
Ah nevermind. The op.scoped_metrics_container is used by Dataflow worker code. Needs to be available.

Reply:
OK. Please add a comment (with a JIRA) here about that.

Reply:
Done. Thanks Robert.